scripts/globals/items/pinion_cesti.lua

```diff
@@ -0,0 +1,33 @@
+-----------------------------------
+-- ID: 20541
+-- Item: Pinion Cesti
+-- Additional Effect: Paralysis
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/magic")
+require("scripts/globals/msg")
+-----------------------------------
+local item_object = {}
+
+item_object.onAdditionalEffect = function(player, target, damage)
+    local chance = 5
+
+    if (VanadielDayOfTheWeek() == xi.day.ICEDAY) then
+        chance = chance+6
+    end
```

Comment:
> [x] that I have read the Contributing Guide

For many many of these scripts:

https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md#no-parentheses-unless-needed-to-clarify-order-of-operations

Reply:
Sorry @zach2good , all of the additional effects items I made were copied from others, so wasn't changed other than values. So, I should remove the () after VanadielDayOfTheWeek? If so, I can work on fixing those.

Reply:
`VanadielDayOfTheWeek` is a function, so you need to call it with `VanadielDayOfTheWeek()`.

Reply:
one should understand the language before attempting to do these things